apps/system/lockscreen/raptor/lockscreen.esp.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = function(filepath, outpath, requiredir) {
+  var opts = {};
+  if (outpath) {
+    var fs = require('fs');
+    opts.writer = {
+      write: function(advice) {
+        var code = advice.code;
+        fs.writeFile(outpath, code);
+      }
+    };
+  }
+  var Espect = require(requiredir + 'espect.js');
+  var espect = new Espect(opts);
```

Comment:
In this case you need to wait the requiredir pass in, which is not available at the top of file.